frontend/src/pages/team/Settings/Devices.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+    <SectionTopMenu hero="Device Provisioning" help-header="Team - Settings - Devices" info="A list of device provisioning tokens. Assign them to devices to facilitate auto enrolment.">
```

Comment:
```suggestion
    <SectionTopMenu hero="Device Provisioning" help-header="Team - Settings - Devices" info="A list of device provisioning tokens that can be used to auto register devices to the team.">
```

We don't use 'enrolment' anywhere else so let's not introduce a new term.

Should we, in the UI, use 'register' more consistently rather than 'provision'? We currently talk about 'auto provisioning' which implies 'provisioning' is a thing - but it isn't.

So should it be 'auto registration'?

Reply:
Addressed in [3040b7a](https://github.com/flowforge/flowforge/pull/1664/commits/3040b7ab585249b218118051d47662bac0ebbaad)